Lib/test/test_capi/test_opt.py

```diff
@@ -1706,7 +1706,7 @@ def f(n):
         self.assertEqual(res, 2 * TIER2_THRESHOLD)
         self.assertIsNotNone(ex)
         uops = get_opnames(ex)
-        self.assertEqual(uops.count("_GUARD_NOS_LIST"), 1)
+        self.assertEqual(uops.count("_GUARD_NOS_LIST"), 0)
```

Comment:
Can you please update the comment above and change `Guarded` to `Unguarded`. The guard has now been removed.

Same for `test_remove_guard_for_known_type_dict`